tastyworks/models/order.py

```diff
@@ -54,15 +59,27 @@ class OrderDetails(object):
     def is_executable(self) -> bool:
         required_data = all([
             self.time_in_force,
-            self.price_effect,
-            self.price is not None,
             self.type,
             self.source
         ])
 
+        if self.type == OrderType.STOP:
+            non_stop_required_data = all([
+                self.price_effect,
```

Comment:
Maybe separate the `self.price_effect` into a separate check because that seems to be a requirement for all types of orders. The price check for STOP vs other order types can then be obviated to something similar to: 
```
if self.type != OrderType.STOP and not self.price:
    return False
```

Thoughts?

Reply:
Also, do we need to check `STOP_LIMIT` types as well?